infra/src/main/java/com/redhat/service/bridge/infra/models/dto/ConnectorStatus.java

```diff
@@ -3,10 +3,16 @@
 // TODO-MC review statuses
 // https://issues.redhat.com/browse/MGDOBR-195
 public enum ConnectorStatus {
-    REQUESTED,
-    PROVISIONING,
-    AVAILABLE,
-    DELETION_REQUESTED,
-    DELETED,
-    FAILED
+    ACCEPTED("accepted"),
```

Comment:
Unfortunately you have to migrate the existing rows in the database otherwise the demo cluster is broken. 
In short you have to do something like https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/fb7d556e761207bd649b1c20ba5d50a5f16c1226/manager/src/main/resources/db/migration/0.1/V0.1.6__migrate_statuses.sql